src/testing/euler_gaussian_bump.cpp

```diff
@@ -40,6 +40,19 @@ EulerGaussianBump<dim,nstate>::EulerGaussianBump(const Parameters::AllParameters
 template<int dim, int nstate>
 int EulerGaussianBump<dim,nstate>
 ::run_test () const
+{
+    int convergence_order_achieved = 0;
+    double run_test1_output = run_test1();
+    if (abs(convergence_order_achieved-run_test1_output) > 1e-15) // If run_test1() returns 1
```

Comment:
add comments about why you're doing this check. explain run_test1 can do enthalpy as well

Reply:
also maybe consider renaming `run_test1` to something more descriptive

Reply:
It's commented and renamed to run_euler_gaussian_bump().